backend_jobs/ingestion_update_visibility/main.py

```diff
@@ -0,0 +1,118 @@
+"""
+  Copyright 2020 Google LLC
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+A pipeline to update the visibility in documents/images from a specific dataset
+by a specific provider/ pipeline run.
+
+The pipeline uses Python's Apache beam library to parallelize the different stages.
+The images are taken from a Firestore database.
+"""
+
+from __future__ import absolute_import
+
+import argparse
+import logging
+import apache_beam as beam
+from apache_beam.options.pipeline_options import PipelineOptions
+
+from backend_jobs.ingestion_update_visibility.pipeline_lib.firestore_database import GetDataset
+from backend_jobs.ingestion_update_visibility.pipeline_lib.firestore_database import \
+    UpdateVisibilityInDatabaseCollection
+from backend_jobs.ingestion_update_visibility.pipeline_lib.firestore_database import \
+    UpdateVisibilityInDatabaseSubcollection
```

Comment:
Do not import the class directly, but import the module instead: https://google.github.io/styleguide/pyguide.html#224-decision

i.e. change the import to 
`from backend_jobs.ingestion_update_visibility.pipeline_lib import firestore_database`
and then use with `firestore_database.UpdateVisibilityInDatabaseSubcollection`